src/ui/components/services/storage.ts

```diff
@@ -0,0 +1,21 @@
+import Todo from '../models/todo';
```

Comment:
Ugh, looks like https://github.com/glimmerjs/glimmer-application-pipeline/blob/master/lib/broccoli/default-module-configuration.ts does not define services as a type. We should chat with @dgeb to see if that was intentional, but for now you can put it in `src/utils/storage.ts`.